build/KoreBuild.sh

```diff
@@ -118,6 +118,15 @@ fi
 
 if [ "$(uname)" == "Darwin" ]; then
     ulimit -n 2048
+
+    # Check that OS is 10.12 or newer
+    osx_version="$(sw_vers | grep ProductVersion | awk '{print $2}')"
```

Comment:
It could, but then I'd have to have a second call to sw_vers and grep to fetch the version for the error message